school_attendance/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Attendance Management",
+    "version": "14.0.1.0.0",
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "website": "http://www.serpentcs.com",
+    "images": ["static/description/SchoolAttendance.png"],
+    "category": "School Management",
+    "license": "AGPL-3",
+    "summary": "A Module For Attendance Management In School",
+    "complexity": "easy",
+    "depends": ["school"],
```

Comment:
Are we sure it has not dependency on other modules? I wonder.

Reply:
Previously, the dependency includes 'hr' module too, but as 'hr' module is also included in the dependency of the 'school' module, removed it from here.